Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -104,7 +104,7 @@ void NANDImporter::FindSuperblock()
 
   m_nand_fat_offset = superblock + 0xC;
   m_nand_fst_offset = m_nand_fat_offset + 0x10000;
-  INFO_LOG(DISCIO, "Using superblock version 0x%x at position 0x%x. FAT/FST offset: 0x%x/0x%x",
+  INFO_LOG(DISCIO, "Using superblock version 0x%x at position 0x%lx. FAT/FST offset: 0x%lx/0x%lx",
```

Comment:
Dang that's too funny, I see this issue, then do a quick google search to fact-check, and then post my comment below, when you've already said the exact same thing. lol